src/components/Header.js

```diff
@@ -0,0 +1,9 @@
+import style from "./Header.module.scss";
+
+export const Header = () => {
+  return (
+    <div className={style.headerStyle}>
+      <h1>RoadToStar</h1>
+    </div>
+  );
+};
```

Comment:
This is called explicit return, in es6 (ecmascript version 6, which was released in 2015 or something like that). When you return a single expression (explanation what expression is [here](https://www.joshwcomeau.com/javascript/statements-vs-expressions/) ) we can use IMPLICIT return.

Like so:
```suggestion
export const Header = () => (
    <div className={style.headerStyle}>
      <h1>RoadToStar</h1>
    </div>
  );
```

Same for classic (non-component) function `const sumTwo = (a, b) => a + b` 

I usually have eslint rule for it, so it is done automatically, I just wanted to explain the difference and the wording around it. In case somebody would mention it

Reply:
Are you sure this is right? When I delete the return statement I am getting this error: "'Header' cannot be used as a JSX component.
  Its return type 'void' is not a valid JSX element."
  
  And the Header is not rendering :( 
  ChatGPT aso things that the return shoudl be there.
  Please advise.

Reply:
done